nautobot_golden_config/views.py

```diff
@@ -292,6 +311,42 @@ def get(self, request, pk, config_type):  # pylint: disable=invalid-name,too-man
                 + "@@"
                 + output[second_occurence + 2 :]
             )
+        elif config_type == "json_compliance":
+            compliance_objects = models.ConfigCompliance.objects.filter(device=device.id)
+            actual = {}
+            intended = {}
+            most_recent_time = datetime(1970, 1, 1, tzinfo=timezone.utc)
```

Comment:
Does the datetime now method not work?

Reply:
See previous comment, want to set to an older time since I'm looping through config compliance rules to find the most recently updated time.